TeamCode/src/main/java/org/firstinspires/ftc/teamcode/opmodes/DrivebaseAuto.java

```diff
@@ -0,0 +1,91 @@
+
+package org.firstinspires.ftc.teamcode.opmodes;
+
+import com.acmerobotics.roadrunner.TrajectoryActionBuilder;
+import com.arcrobotics.ftclib.command.CommandScheduler;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.acmerobotics.roadrunner.Action;
+import com.acmerobotics.roadrunner.Pose2d;
+import com.acmerobotics.roadrunner.SequentialAction;
+import com.acmerobotics.roadrunner.Vector2d;
+import com.acmerobotics.roadrunner.ftc.Actions;
+import org.firstinspires.ftc.teamcode.roadrunner.MecanumDrive;
+
+import org.firstinspires.ftc.teamcode.*;
+
+@Autonomous(name="Drive-Base-Auto", group = "Real")
```

Comment:
You are allowed to use spaces. It could be "Drive Base Auto." Anyways, just copy paste all my comments about the last auto mode and they basically all apply here too.

Reply:
the Drive Base Auto has been trashed now :(((

Reply:
Oh okay...